plugins/signals-node/api-report.md

```diff
@@ -16,15 +16,15 @@ export class DefaultSignalService implements SignalService {
 
 // @public (undocumented)
 export type SignalPayload = {
-  recipients: string[] | null;
+  receivers: string[] | string | null;
```

Comment:
Hmm, not sure I like receivers more but not gonna stress it for now

Reply:
XD

Reply:
Hmm, should we switch it back and change the BEP? Feels like that's where we're going anyway